src/main/kotlin/com/cognifide/gradle/aem/pkg/DownloadTask.kt

```diff
@@ -0,0 +1,121 @@
+package com.cognifide.gradle.aem.pkg
+
+import com.cognifide.gradle.aem.api.AemConfig
+import com.cognifide.gradle.aem.api.AemTask
+import com.cognifide.gradle.aem.base.vlt.VltFilter
+import com.cognifide.gradle.aem.instance.Instance
+import com.cognifide.gradle.aem.instance.InstanceSync
+import com.cognifide.gradle.aem.internal.PropertyParser
+import com.cognifide.gradle.aem.internal.file.FileContentReader
+import org.apache.commons.io.FilenameUtils
+import org.gradle.api.file.DuplicatesStrategy
+import org.gradle.api.tasks.InputDirectory
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.Nested
+import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.bundling.Zip
+import java.io.File
+
+open class DownloadTask : Zip(), AemTask {
+
+    companion object {
```

Comment:
usually companion objects are below the class. I am also fighting with myself to remember it ;)

Reply:
It depends on the use case? It's more or less a static container. For former java dev this is more readable

Reply:
yep, but code written by Jetbrains usually has it at down. I am trusting experts ;)